pages/app/index.tsx

```diff
@@ -22,12 +22,18 @@ interface GlobalFlags {
   appLayoutWidget?: boolean;
   appLayoutToolbar?: boolean;
 }
+// used for local dev / testing
+interface CustomFlags {
+  appLayoutDelayedWidget?: boolean;
+}
```

Comment:
Example of this flag in action: https://d21d5uik3ws71m.cloudfront.net/components/0910fc65859f795dbdd5b95eeebd2b6809daa93a/dev-pages/index.html#/light/app-layout/default?appLayoutWidget=true&appLayoutDelayedWidget=true

Notice how the side panels loaded with a delay